ironfish-zkp/Cargo.toml

```diff
@@ -29,6 +29,8 @@ ff = "0.12.0"
 group = "0.12.0"
 jubjub = { git = "https://github.com/iron-fish/jubjub.git", branch = "blstrs" }
 lazy_static = "1.4.0"
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
This feature should not be generally available for anyone, see the [getrandom documentation](https://docs.rs/getrandom/latest/getrandom/#webassembly-support):

> This feature should only be enabled for binary, test, or benchmark crates. Library crates should generally not enable this feature, leaving such a decision to *users* of their library. Also, libraries should not introduce their own `js` features just to enable `getrandom`’s `js` feature.

One of the reasons for that is that WASM may be used in environments other than a web browser and Node.js, and this feature restricts usage.

Reply:
Yes, but it is specially prepared for wasm, if we don't enable it, there will be compilation problems